js/ckeditor5_plugins/plainPaste/src/plainPaste.js

```diff
@@ -24,8 +24,29 @@ export default class PlainPaste extends Plugin {
 
     editingView.document.on('clipboardInput', (evt, data) => {
       const dataTransfer = data.dataTransfer;
-      let content = plainTextToHtml(dataTransfer.getData('text/plain'));
-      data.content = this.editor.data.htmlProcessor.toView(content);
+      const html = dataTransfer.getData('text/html');
+
+      const cleanHtml = sanitizeHtml(html, {
+        allowedTags: [
+          'p',
+          'a',
+          'h2',
+          'h3',
+          'h4',
+          'h5',
+          'ul',
+          'ol',
+          'li',
+          'table',
+          'thead',
+          'tbody',
+          'th',
+          'tr',
+          'td',
+        ],
```

Comment:
div, span, b, i, strong, em, dl, blockquote, code?

Reply:
For now I wanna try to be pretty heavy handed. This is only when pasting. I definitely don't want to do div tags but maybe a few of those others wouldn't be bad.

Reply:
I'd argue that if you're going to include `dl`, you also need `dt` and `dd`.